docs/man_pages/index.md

```diff
@@ -36,6 +36,11 @@ Command | Description
 [plugin](lib-management/plugin.html) | Lets you manage the plugins for your project.
 [livesync](project/testing/livesync.html) | Synchronizes the latest changes in your project to devices.
 
+## Publishing Commands
+Command | Description
+---|---
+[appstore](project/publishing/appstore.html) | Lists applications registered in iTunes Connect.
```

Comment:
Consider adding the appstore upload command as well here as it does something completely different.